### ISSUE TITLE
🌈 Colors 🌈

### ISSUE THREAD

**Issue by sindresorhus** [SamVerschueren/dev-time-cli#12]
Give the output some colors.

**Comment by SamVerschueren** [comment]
Not much we can colorize though :). Or do you want a fancy rainbow style nyan cat kinda output?

**Comment by sindresorhus** [comment]
You could give some style to the names and time, and the spinner.

**Comment by SamVerschueren** [comment]
Good idea! If someone comes up with a nice color scheme, PR welcome :).
